src/Components/Web.JS/src/Rendering/RendererEventDispatcher.ts

```diff
@@ -0,0 +1,18 @@
+import { EventDescriptor } from './BrowserRenderer';
+import { UIEventArgs } from './EventForDotNet';
+
+type EventDispatcher = (eventDescriptor: EventDescriptor, eventArgs: UIEventArgs) => Promise<void>;
+
+let eventDispatcherInstance: EventDispatcher;
+
+export function eventDispatcher(eventDescriptor: EventDescriptor, eventArgs: UIEventArgs): Promise<void> {
```

Comment:
A more natural name might be `dispatchEvent`, since it actually does the thing rather than (say) returning a function that does it.

If possible could we also make this return `void` as per the other comment above? Just to be clear that the usage pattern is fire-and-forget.

Reply:
I'll make everything void returning